conjur/controller/init_controller.py

```diff
@@ -42,22 +42,28 @@ def load(self):
         """
         Method that facilitates all method calls in this class
         """
+        if self.conjurrc_data.conjur_url is None:
+            self.get_conjur_server_url()
```

Comment:
extracted the two (format/validate) to their own method because I need the URL formatted both for when the user inputs the URL and when they didn't